polaris-migrator/src/migrations/v11-styles-replace-custom-property-legacy/v11-styles-replace-custom-property-legacy.ts

```diff
@@ -0,0 +1,38 @@
+import type {FileInfo, API} from 'jscodeshift';
+
+import stylesReplaceCustomProperty from '../styles-replace-custom-property/styles-replace-custom-property';
+
+const replacementMaps = {
+  '/.+/': {
+    '--p-override-loading-z-index': '--p-z-index-6',
+    '--p-choice-size': '20px',
+    '--p-icon-size-small': '8px',
+    '--p-icon-size-medium': '20px',
+    '--p-choice-margin': '--p-space-025',
+    '--p-control-border-width': '--p-border-radius-05',
+    '--p-banner-border-default':
+      'inset 0 (--p-border-width-1) 0 0 var(--p-color-border-strong), inset 0 0 0 (--p-border-width-1) var(--p-color-border-strong)',
+    '--p-banner-border-success':
+      'inset 0 (--p-border-width-1) 0 0 var(--p-color-border-success-subdued), inset 0 0 0 (--p-border-width-1) var(--p-color-border-success-subdued)',
+    '--p-banner-border-highlight':
+      'inset 0 (--p-border-width-1) 0 0 var(--p-color-border-info-subdued), inset 0 0 0 (--p-border-width-1) var(--p-color-border-info-subdued)',
+    '--p-banner-border-warning':
+      'inset 0 (--p-border-width-1) 0 0 var(--p-color-border-caution-subdued), inset 0 0 0 (--p-border-width-1) var(--p-color-border-caution-subdued)',
+    '--p-banner-border-critical':
+      'inset 0 (--p-border-width-1) 0 0 var(--p-color-border-critical-subdued), inset 0 0 0 (--p-border-width-1) var(--p-color-border-critical-subdued)',
```

Comment:
@heyjoethomas Are the following tokens v11 values? 

- `--p-color-border-strong`
- `--p-color-border-success-subdued`
- `--p-color-border-info-subdued`
- `--p-color-border-caution-subdued`
- `--p-color-border-critical-subdued`